src/main/java/ru/yandex/practicum/filmorate/storage/user/UserDbStorage.java

```diff
@@ -0,0 +1,133 @@
+        }
+        return user;
+    }
+
+    @Override
+    public User updateUser(User newUser) {
+        cheсkUserPresence(newUser.getId());
+        String sql = "UPDATE USERS SET email=?, login=?, name=?, birthday=? WHERE id=?";
+        jdbcTemplate.update(
+                sql,
+                newUser.getEmail(),
+                newUser.getLogin(),
+                newUser.getName(),
+                Date.valueOf(newUser.getBirthday()),
+                newUser.getId()
+        );
+        return newUser;
+    }
+
+    @Override
+    public List<User> getAllUsers() {
+        String sql = "SELECT * FROM USERS";
+        List<User> users = jdbcTemplate.query(sql, new UserRowMapper());
+        return users;
+    }
+
+    @Override
+    public List<User> getCommonFriends(int userOneId, int userTwoId) {
+        cheсkUserPresence(userOneId);
+        cheсkUserPresence(userTwoId);
+        String sql = """
+                SELECT * FROM USERS u
+                JOIN friends f1 ON u.id = f1.friend_id AND f1.user_id = ?
+                JOIN friends f2 ON u.id = f2.friend_id AND f2.user_id = ?
+                """;
+        List<User> commonFriends = jdbcTemplate.query(sql, new UserRowMapper(), userOneId, userTwoId);
+        return commonFriends;
+    }
+
+    @Override
+    public void deleteUser(int id) {
+        cheсkUserPresence(id);
+        String sql = "DELETE FROM USERS WHERE id = ?";
+        jdbcTemplate.update(sql, id);
+    }
+
+    @Override
+    public User getUserById(int userId) {
+        cheсkUserPresence(userId);
+        String sql = "SELECT * FROM USERS WHERE id = ?";
+        return jdbcTemplate.queryForObject(sql, new UserRowMapper(), userId);
+    }
+
+    @Override
+    public void addFriend(int userId, int frienId) {
+        cheсkUserPresence(userId);
+        cheсkUserPresence(frienId);
+        String sql = "INSERT INTO friends (user_id, friend_id, status) VALUES (?, ?, ?)";
+        jdbcTemplate.update(sql, userId, frienId, true);
+    }
+
+    @Override
+    public void deleteFriend(int userId, int friendId) {
+        cheсkUserPresence(userId);
+        cheсkUserPresence(friendId);
+        String sql = "DELETE FROM friends WHERE user_id = ? AND friend_id = ?";
+        jdbcTemplate.update(sql, userId, friendId);
+    }
+
+    @Override
+    public List<User> getAllFriends(int userId) {
+        cheсkUserPresence(userId);
+        String sql = "SELECT * FROM users JOIN friends ON users.id = friends.friend_id WHERE friends.user_id = ?";
+        return jdbcTemplate.query(sql, new UserRowMapper(), userId);
+    }
+
+    @Override
+    public boolean isFriend(int userId, int friendId) {
+        String sql = "SELECT * FROM friends WHERE user_id = ? AND friend_id = ?";
+        SqlRowSet rowSet = jdbcTemplate.queryForRowSet(sql, userId, friendId);
+        return rowSet.next();
+    }
+
+    private void cheсkUserPresence(int userId) {
```

Comment:
В имени метода присутствует кириллическая `«с»`
```suggestion
    private void checkUserPresence(int userId) {
```